packages/webpack-plugin/src/index.js

```diff
@@ -28,19 +29,36 @@ class LoadablePlugin {
       chunkGroupChildren: true,
       hash: true,
       ids: true,
+      modules: true,
```

Comment:
this _may_ affect stats size and time required to collect them.
Ideally, extra functionality should be opt-in, not opt-out.

Reply:
Not stats size, but time to collect - yes. These modules are later deleted, after I parse them with federatedStatsCollector. Not sure how much time is lost on module collection, but if it's worth it in the end, I believe it is possible to overcomplicate the plugin to test if wmf plugin is also connected. And if it is - enabling module parsing.